pkg/kn/commands/revision/revision_list_test.go

```diff
@@ -75,9 +77,11 @@ func TestRevisionListEmptyByName(t *testing.T) {
 }
 
 func TestRevisionListDefaultOutput(t *testing.T) {
-	revision1 := createMockRevisionWithParams("foo-abcd", "foo")
-	revision2 := createMockRevisionWithParams("bar-wxyz", "bar")
-	RevisionList := &v1alpha1.RevisionList{Items: []v1alpha1.Revision{*revision1, *revision2}}
+	revision1 := createMockRevisionWithParams("foo-abcd", "foo", "1")
+	revision2 := createMockRevisionWithParams("bar-abcd", "bar", "1")
+	revision3 := createMockRevisionWithParams("foo-wxyz", "foo", "2")
+	revision4 := createMockRevisionWithParams("bar-wxyz", "bar", "2")
```

Comment:
Could you please also add a revision with generation "10" to catch the sorting issue with strings ?